next-level/lib/checkers/swift.py

```
@@ -0,0 +1,95 @@
+            result["formatted"] = proc.returncode == 0
+        except (subprocess.TimeoutExpired, FileNotFoundError):
+            pass
+
+    # Strip unnecessary comments
+    try:
+        from comment_stripper import strip_comments
+        strip_result = strip_comments(filepath, "swift")
+        result["comments_stripped"] = strip_result.get("stripped", 0)
+    except ImportError:
+        result["comments_stripped"] = 0
+    except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)